src/cluster.c

```diff
@@ -195,7 +195,7 @@ int clusterLoadConfig(char *filename) {
                     server.cluster->lastVoteEpoch =
                             strtoull(argv[j+1],NULL,10);
                 } else {
-                    serverLog(LL_WARNING,
+                    serverLog(LL_NOTICE,
```

Comment:
oran: i'm not sure about this one

Reply:
Yes, this doesn't look like a warning since it's technically allowed to be divergent. I don't feel strongly either way about this one though.

Reply:
ok, decide change it to notice